lib/Mojolicious/Plugin/OpenAPI/Security.pm

```diff
@@ -1,19 +1,23 @@
 package Mojolicious::Plugin::OpenAPI::Security;
 use Mojo::Base -base;
 
+my %SECURITY_DEFINITION_PATH
```

Comment:
Maybe just call it %DEF_PATH or something to make the line length shorter? We're already inside Security.pm, so starting with "SECURITY_" is a bit redundant.